src/main/resources/i18n/Resources_es.properties

```diff
@@ -92,8 +78,6 @@ cloudAlbumsProviderProgressTitle=Cargando albumes en Google Photos
 uploaderAlbumProgressTitle=Añadiendo a los álbumes
 uploaderFileProgressTitle=Subiendo archivos multimedia
 albumManagerProgressStatusTitle=Reconciliando álbumes con Google Photos
-# this is the label on top of the failures table; this table is accessible when there are upload failures;
```

Comment:
These comments were not TODOs, they just explained the meaning of this phrase. If not too difficult, please revert these two lines, i.e. restore the two comments.